src/config/data.ts

```diff
@@ -289,6 +299,77 @@ const confirmationModalText = {
   deleteProfile: 'Do you really want to delete your profile?',
 };
 
+const technologiesList = [
+  {
+    id: 0,
+    title: 'React',
+    imgSrc: reactImg,
+    link: 'https://beta.reactjs.org/',
+  },
+  {
+    id: 1,
+    title: 'TypeScript',
+    imgSrc: tsImg,
+    link: 'https://www.typescriptlang.org/',
+  },
+  {
+    id: 2,
+    title: 'React Router',
+    imgSrc: routerImg,
+    link: 'https://reactrouter.com/',
+  },
+  {
+    id: 3,
+    title: 'Vite',
+    imgSrc: viteImg,
+    link: 'https://vitejs.dev/',
+  },
+  {
+    id: 4,
+    title: 'React Bootstrap',
+    imgSrc: bootstrapImg,
+    link: 'https://react-bootstrap.github.io/',
+  },
+  {
+    id: 5,
+    title: 'React Hook Form',
+    imgSrc: formImg,
+    link: 'https://react-hook-form.com/',
+  },
+  {
+    id: 6,
+    title: 'React Beautiful DnD',
+    imgSrc: dndImg,
+    link: 'https://github.com/atlassian/react-beautiful-dnd',
+  },
+];
+
+const developersList = [
+  {
+    id: 0,
+    name: 'Stanislav Kravchuk',
+    imgSrc: stasImg,
+    descr: 'Created adaptive header & burger menu, welcome page, loading spinner',
+    link: 'https://github.com/kravchuk-st',
+  },
+  {
+    id: 1,
+    name: 'Philipp Khromov',
```

Comment:
Don't forget to add these names under photos

Reply:
done